protocol/app/upgrades/v5.0.0/upgrade_container_test.go

```diff
@@ -120,6 +137,20 @@ func preUpgradeCheckPerpetualMarketType(node *containertest.Node, t *testing.T)
 	}
 }
 
+func postUpgradeCheckVoteExtensions(node *containertest.Node, t *testing.T) {
+	consensusParams := &consensus.QueryParamsResponse{}
+	resp, err := containertest.Query(
+		node,
+		consensus.NewQueryClient,
+		consensus.QueryClient.Params,
+		&consensus.QueryParamsRequest{},
+	)
+	require.NoError(t, err)
+	err = proto.UnmarshalText(resp.String(), consensusParams)
+	require.NoError(t, err)
+	assert.True(t, consensusParams.Params.Abci.VoteExtensionsEnableHeight > int64(0))
```

Comment:
Shld this not be checking that the `VoteExtensionEnableHeight == height_of_upgrade + 1`

Reply:
+ 1

Reply:
It was not clear to me how to get the height of the upgrade via these container test objects. Can see if it's doable though.

Reply:
Updated.